app/views/payments/_payment.html.haml

```diff
@@ -0,0 +1,34 @@
+= semantic_form_for(@payment, url: conference_payments_path) do |f|
+  .form-group
+    = f.label :first_name, 'First name(as on card)'
+    = f.text_field :first_name, class: "form-control", placeholder: "John", id: "first_name"
+  .form-group
+    = f.label :last_name, 'Last name(as on card)'
+    = f.text_field :last_name,class: "form-control", placeholder: "Doe", id: "last_name"
+  %span.pull-right
+    %img.img-responsive{:src => image_url('credit_card.png')}
+  .form-group
+    = f.label :credit_card_number, "Credit Card number(without spaces)"
+    .input-group
+      = f.text_field :credit_card_number, class: "form-control", placeholder: "XXXX XXXX XXXX XXXX", id: "credit_card_number"
+      %span.input-group-addon
+        %i.fa.fa-credit-card
+  .form-group.col-md-6
+    = f.label :expiration_month
+    = f.select :expiration_month, months, {}, class: "form-control", id: "expiration_month"
+  .form-group.col-md-6
+    = f.label :expiration_year
+    = f.select :expiration_year, years, {}, class: "form-control", id: "expiration_year"
+  .form-group.col-md-10
+    = f.label :card_verification_value, 'Security Code(3 on back, AmEx: 4 on front)'
+    = f.text_field :card_verification_value, class: "form-control", placeholder: "XXX", id: "card_verification_value"
+    = f.number_field :amount, value: @total_amount_to_pay, class: "form-control", type: 'hidden'
+  %ul.price-tags.pull-right
+    %li.text-muted
+      you will pay
+    %li
+      %a
+        = number_to_currency @total_amount_to_pay
```

Comment:
I like this "you will pay" ribbon, but I think it should be AFTER the "charge card" button..

![screenshot from 2016-07-21 13-23-23](https://cloud.githubusercontent.com/assets/2944/17021168/6fc1dd3c-4f46-11e6-827d-551bae5a6c51.png)